integration-tests/test_client.py

```diff
@@ -115,3 +115,24 @@ def test_insights_details_file_exists(insights_client):
     insights_client.run("--check-results")
     # Verify that insights-details.json gets re generated
     assert os.path.isfile(output_file)
+
+
+@pytest.mark.usefixtures("register_subman")
+def test_insights_directory_files(insights_client):
+    """
+    Test that /var/lib/insights have content available
+    """
+    directory = "/var/lib/insights"
+    registered_contents = [
+        "last_stable.egg",
+        "last_stable.egg.asc",
+    ]
```

Comment:
Should this test also consider files like
- `/etc/insights-client/.lastupload`
- `/etc/insights-client/.registered`
 and so on? Those are always present on registered system as well.

Reply:
Yeah but they are not in this directory, this test should focus just on that IMHO